tutorial/create-a-simple-squid.md

```diff
@@ -47,7 +47,15 @@ node -r dotenv/config lib/processor.js
 npx squid-graphql-server
 ```
 
-Bear in mind this is not strictly **necessary**, but it is always useful to check that everything is in order. If you are not interested, you could at least get the Postgres container running with `docker compose up -d`.
+Bear in mind this is not strictly **necessary**, but it is always useful to check that everything is in order.
+
+{% hint style="info" %}
+These commands are supposed to be run the first time, right after cloning the template.
+
+Some, like `npx sqd db create`,  may throw an error, because the database had already been previously created.
```

Comment:
```suggestion
Some, like `npx sqd db create`,  may throw an error, because the database had already been previously created. In this case just run `npx sqd db drop` and run `npx sqd db create` again.
```